### PR TITLE
Add ingress/egress TCP database connection to default security group

### PR DESCRIPTION
**Pull request by tnation14**
Modified the default database security group to include ingress/egress TCP connections on the database port. 
## Testing

Test using [azavea/raster-foundry-deployment#2](https://github.com/azavea/raster-foundry-deployment/pull/2).

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -4,6 +4,19 @@
 
 resource "aws_security_group" "postgresql" {
   vpc_id = "${var.vpc_id}"
+  ingress {
```
Comment (hectcastro) [inline main.tf]: I should have looked at this closer before. I don't think we want to make the assumption that module consumers will want their database open to the VPC. We got away from those assumptions in https://github.com/azavea/terraform-aws-postgresql-rds/pull/14.

**Comment by hectcastro** [comment]
I think keeping `database_port` with 5432 as the default port is worth keeping though.

**Comment by tnation14** [comment]
Made some corrections, can you take a look to confirm?

File: main.tf [hunk 2]
```diff
@@ -1,5 +1,7 @@
 #
+
```
Comment (hectcastro) [inline main.tf]: This is my biggest complaint with the `terraform fmt` subcommand. I sometimes try to remove the extra space it introduces, but not a big deal.

File: README.md [hunk 3]
```diff
@@ -40,9 +39,11 @@ module "postgresql_rds" {
 
 ## Variables
 
+- `vpc_id` - ID of VPC meant to house database
+
```
Comment (hectcastro) [inline README.md]: Extra space here makes the rendered output look a little strange:

terraform-aws-postgresql-rds_readme_md_at_ded6818337164b1377dc3820505b6a6e135b2401_ _azavea_terraform-aws-postgresql-rds

**Comment by hectcastro** [comment]
👍

**Pull request closed without merging**